src/modules/75-cf/pages/feature-flags/components/StaleFlagActions/StaleFlagActions.tsx

```diff
@@ -93,55 +87,18 @@ export const StaleFlagActions: FC<StaleFlagActionsProps> = ({ flags, onAction })
                 )}
               </>
             )}
-            <Button
-              padding={{ left: 0 }}
-              text={getString('cf.staleFlagAction.learnMore')}
-              variation={ButtonVariation.LINK}
-              onClick={() => {
-                setShowInfo(true)
-              }}
-            />
-          </Layout.Horizontal>
-          {showInfo && (
-            <Drawer
-              onClose={hideShowInfo}
-              usePortal
-              autoFocus
-              canEscapeKeyClose
-              canOutsideClickClose
-              enforceFocus={false}
-              hasBackdrop
-              isOpen
-              position={Position.RIGHT}
-              size={Drawer.SIZE_STANDARD}
-            >
-              <Button
-                className={css.closeButton}
-                icon="main-close"
-                iconProps={{ size: 12 }}
-                onClick={hideShowInfo}
-                aria-label={getString('close')}
-              />
-              <Layout.Vertical className={css.infoDrawer} height={'100%'} padding="xxlarge">
-                <Heading level={4} className={css.infoTitle} font={{ variation: FontVariation.H4 }}>
-                  {getString('cf.staleFlagAction.flagLifecycleExplained')}
-                </Heading>
-                <Text
-                  className={css.infoText}
-                  font={{ variation: FontVariation.BODY }}
-                  color={Color.GREY_500}
-                  padding="medium"
-                >
-                  {getString('cf.staleFlagAction.flagLifecycleDesc')}
+            <Layout.Horizontal spacing="medium" flex={{ alignItems: 'center' }}>
+              <a
+                href={'https://developer.harness.io/docs/feature-flags/ff-creating-flag/manage-stale-flags/'}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <Text padding={{ left: 0 }} font={{ variation: FontVariation.BODY }} color={Color.PRIMARY_7}>
+                  {getString('cf.staleFlagAction.learnMore')}
                 </Text>
-                <Layout.Horizontal padding="medium">
-                  <Button className={css.gotItButton} variation={ButtonVariation.PRIMARY} onClick={hideShowInfo}>
-                    {getString('common.gotIt')}
-                  </Button>
-                </Layout.Horizontal>
-              </Layout.Vertical>
-            </Drawer>
-          )}
+              </a>
+            </Layout.Horizontal>
```

Comment:
This might do the trick...
```suggestion
          <Container flex={{ alignItems: 'center' }}>
            <a
              href="https://developer.harness.io/docs/feature-flags/ff-creating-flag/manage-stale-flags/"
              target="_blank"
              rel="noopener noreferrer"
            >
              <Text inline font={{ variation: FontVariation.BODY }} color={Color.PRIMARY_7}>
                {getString('cf.staleFlagAction.learnMore')}
              </Text>
            </a>
          </Container>
```

Reply:
thanks have updated